mica/archive/aca_dark/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .dark_cal import *
-from .dark_model import *
+from chandra_aca.dark_model import *
```

Comment:
I don't see a way to get "from mica.archive.aca_dark.dark_model import get_warm_fracs" to work from the __init__.py, so I just added back a light dark_model.py .